README.md

```diff
@@ -140,6 +140,34 @@ module.exports = {
 
 These JSDoc custom tags are used to adjust generated pact interactions.
 
+<a name="pact-axios"></a>
+`@pact-axios` - sets REST method, expected body for the current response, expected body for current request and query based on axios definitions
```

Comment:
Firstly I would not divide interactions definitions into only two - because you can mix them, for example very frequent case while testing pact-axios was making POST request with different response-status then the one which would be created as default from method. 
Secondly creating  `@pact source=axios` makes axios annotation less flexible for extension like adding some axios related options which would make `@pact` do too many things.
Thirdly I'm not convinced to changing annotations API - right now they are very consistent so once you learned that firstly you set option using @pact-XXXX and then passing necessary data it will work for all options in the same way.